### DISCUSSION TITLE
blog/2023/01/07/why-not-rust-for-omnigres/

### DISCUSSION THREAD

**Discussion by giscus** [yrashk/yrashk.github.io#3] [General]
# blog/2023/01/07/why-not-rust-for-omnigres/

Writings on software development and work practices, entrepreneurship, tech in general.

https://yrashk.com/blog/2023/01/07/why-not-rust-for-omnigres/

**Comment by Jeevananthan-23** [comment]
Have you checkout the DBOS/Apiary(FAAS) which is developed in `Java` and supports VoltDb, Postgres, Mongo, MySQL and Elastic search. I can understand the complexity of the project grows with belong to the other extensions but the features is really good.

 Why not consider Golang in this type of work load I knew **Rust Vs Go** is not going to end and both the langs moving forward getting more improvements. 

After skimming through **DBOS/APIARY** code base it could be great to have this kind of FAAS application platform. `Golang` is good candidate for developing simple and better concurrency application in cloud.
